pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -82,7 +82,7 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 	csiNode, err := pl.csiNodeLister.Get(node.Name)
 	if err != nil {
 		// TODO: return the error once CSINode is created by default (2 releases)
-		klog.V(5).Infof("Could not get a CSINode object for the node: %v", err)
+		klog.V(5).Infof("Could not get a CSINode object for the node", "node", node.Name, "err", err)
```

Comment:
I think we can use ErrorS here.

@cofyc this should not happen now that we are creating CSINode by default, correct?

Reply:
yes, we should return error now